index.html

```diff
@@ -55,6 +55,114 @@ <h1>Hi there I'm Carlos Vivas <br> <span class="highlight">I'm a software Develo
 			</div>
 			<div class="scroll-down-btn"><a href="#"><i class="fa-solid fa-chevron-down"></i></a></div>
 		</section>
+
+		<!-- "Works" section -->
+
+		<section id="works">
+			<header>
+				<h2>My Recent Works</h2>
+				<div class="divider"></div>
+			</header>
+
+			<article class="card">
+				<!-- <img class="card-img" src="assets/works/Img Plaholder.jpg" alt="preview"> -->
+				<div class="card-body">
+					<h3>Multi-Post Stories Gain+Glory 01</h3>
+					<ul>
+						<li>Ruby on rails</li>
+						<li>css</li>
+						<li>JavaScript</li>
+						<li>html</li>
+					</ul>
+					<button disabled="disabled">See Project</button>
+				</div>
+			</article>
+
+			<article class="card">
+				<!-- <img class="card-img" src="assets/works/Img Plaholder.jpg" alt="preview"> -->
+				<div class="card-body">
+					<h3>Multi-Post Stories Gain+Glory 02</h3>
+					<ul>
+						<li>Ruby on rails</li>
+						<li>css</li>
+						<li>JavaScript</li>
+						<li>html</li>
+					</ul>
+					<button disabled="disabled">See Project</button>
```

Comment:
- [ ] Nice working on your `My Recent Works Section`, kindly change the color of the `See Project Button` so it matches what is in the Figma Template.

### Figma Design
<img width="277" alt="fig123" src="https://user-images.githubusercontent.com/48371156/176800753-86a114c1-9051-4cbc-9a84-a220f086a535.png">

### Your Design 
<img width="263" alt="my123" src="https://user-images.githubusercontent.com/48371156/176800811-3f040dd9-2e81-4739-8ed7-0efba804bf42.png">

Reply:
### I'm using an alternative colour palette. the blue one

![imagen](https://user-images.githubusercontent.com/34493013/176801950-8a0cc97a-de88-4ba7-92c0-57ffaede5048.png)